mvp-common/src/test/java/com/hannesdorfmann/mosby/mvp/MvpBasePresenterTest.java

```diff
@@ -35,4 +39,32 @@ public void testOnDestroy(){
   private void noViewAttached(){
     Assert.assertFalse(isViewAttached());
   }
+
+  @Test
+  public void testAttachView() throws Exception {
+    MvpBasePresenter<MvpView> presenter = new MvpBasePresenter<>();
+    assertFalse(presenter.isViewAttached());
+    presenter.attachView(new MvpView() {
+    });
+    assertTrue(presenter.isViewAttached());
+  }
+
+  @Test
+  public void testGetView() throws Exception {
+    MvpBasePresenter<MvpView> presenter = new MvpBasePresenter<>();
+    MvpView mvpView = new MvpView() {
+    };
+    presenter.attachView(mvpView);
+    assertEquals(mvpView, presenter.getView());
+  }
+
+  @Test
+  public void testDetachView() throws Exception {
+    MvpBasePresenter<MvpView> presenter = new MvpBasePresenter<>();
+    MvpView mvpView = new MvpView() {
+    };
+    presenter.attachView(mvpView);
+    presenter.detachView(false);
+    assertEquals(null, presenter.getView());
+  }
```

Comment:
Your intention to add these 3 tests is great! 
But from my point of view all of them are already tested `testOnDestroy()`. Don't you agree?

However, it would be a good idea to rename that method name because it's misleading.
You are right: `assertEquals(null, presenter.getView());`. This test is missing in `testOnDestroy()`